packages/web-player-component/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@eyevinn/web-player-component",
+  "version": "1.0.0",
+  "description": "> TODO: description",
+  "author": "SebastianLjungman <sebastian.ljungman94@gmail.com>",
```

Comment:
Change author to be `Eyevinn Technology <work@eyevinn.se>` Add your personal info to contributions section instead. Please also use the Eyevinn email address